### ISSUE TITLE
nvcc fatal   : Unsupported gpu architecture 'compute_75'

### ISSUE THREAD

**Issue by ruc98** [erikwijmans/Pointnet2_PyTorch#91]
When I run the 'python setup.py build_ext --inplace' command, I get the following error.
```
nvcc fatal   : Unsupported gpu architecture 'compute_75'
error: command '/usr/local/cuda-8.0/bin/nvcc' failed with exit status 1
```
System Specifications:
Driver Version: 440.31
CUDA Version: 10.2
GPU: GTX 1080Ti and RT 2080Ti
Ubuntu version: 16.04.3 LTS (GNU/Linux 4.15.0-76-generic x86_64)
Pytorch version: '1.3.1'

Did anyone else face the same error?
Please help me to fix this error at the earliest.
Thank you.

**Comment by erikwijmans** [comment]
RTX 2080Ti requires CUDA>=10.0

**Comment by ruc98** [comment]
Solved by downgrading CUDA to 10.0 and cudnn=7.5.1
Thanks
